media-proxy/src/mtl.c

```diff
@@ -1270,13 +1270,11 @@ int tx_st20p_shm_init(tx_session_context_t* tx_ctx, memif_ops_t* memif_ops)
     }
 
     /* Set application name */
-    strncpy(tx_ctx->memif_socket_args.app_name, memif_ops->app_name,
-        sizeof(tx_ctx->memif_socket_args.app_name) - 1);
+    strncpy(tx_ctx->memif_socket_args.app_name, memif_ops->app_name, sizeof(tx_ctx->memif_socket_args.app_name) - 1);
```

Comment:
I suppose these `strncpy` cases could be replaced with `strlcpy` which has a better implementation https://linux.die.net/man/3/strlcpy. This is for the future refactoring. I'm not requesting to update this PR

Reply:
I did not know that, thanks for the notice. There are lots of issues that need to be resolved and string copy is one of them.